deployment/lib/lambdas/cf-url-rewriter.ts

```diff
@@ -0,0 +1,70 @@
+import { CloudFrontRequestEvent, CloudFrontRequest } from 'aws-lambda';
+
+const https = require('https');
+
+export async function handler(event: CloudFrontRequestEvent, context, callback): Promise<CloudFrontRequest> {
+  const request = event.Records[0].cf.request;
+
+  console.log('request', JSON.stringify(request));
+
+  if (request.uri.includes("latest")) {
+
+    const indexUri = request.uri.replace(/latest.*/, 'dist/index.json');
+
+    const data = await httpGet('https://' + request.headers.host[0].value + indexUri);
+
+    const redirectResponse = {
+      status: '302',
+      statusDescription: 'Moved temporarily',
+      headers: {
+        'location': [{
+          key: 'Location',
+          value: request.uri.replace('latest', data.latest),
```

Comment:
If the JSON is missing `latest` we just strip it?

Reply:
Added a check for `data && data.latest`